src/utils/storageFacade.utils.ts

```diff
@@ -44,12 +129,12 @@ export async function readPackage(packageRef: string, sessionId: number) {
     logger.error(error.message);
     return null;
   } else if (data) {
-    if (data.length == 0) {
+    if (data.length < 1) {
       //No entry found
       logger.error('Storage Facade: Package "' + packageRef + '" not found');
       return null;
     } else if (data.length > 1) {
-      //Duplicates (how should we handle?)
+      //Duplicates in DB(how should we handle? for now just log it)
```

Comment:
perhaps check if vulnerabilities/junction are also duplicated and then call delete function?